lib/credit_card_sanitizer.rb

```diff
@@ -44,13 +44,16 @@ class CreditCardSanitizer
   NONEMPTY_LINE_NOISE = /#{LINE_NOISE_CHAR}{1,5}/
   SCHEME_OR_PLUS = /((?:&#43;|\+)|(?:[a-zA-Z][\-+.a-zA-Z\d]{,9}):[^\s>]+)/
   NUMBERS_WITH_LINE_NOISE = /#{SCHEME_OR_PLUS}?\d(?:#{LINE_NOISE}\d){10,30}/
+  HTML_TAGS = /<\D[^>]*>/
```

Comment:
This regex doesn't look half bad, and it was from the book Mastering Regular Expressions:
http://haacked.com/archive/2004/10/25/usingregularexpressionstomatchhtml.aspx/